plugins/i18n.ts

```diff
@@ -16,6 +16,7 @@ const findCurrencyBasedOnMagentoStoreCode = (storeCode: string, locales: Array<s
  * @param currency {string}
  * @returns {string}
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we disable eslint instead of fixing the issue?

Reply:
Fixed.